weaviate/config.py

```diff
@@ -7,7 +7,7 @@
 @dataclass
 class ConnectionConfig:
     session_pool_connections: int = 20
-    session_pool_maxsize: int = 20
+    session_pool_maxsize: int = 100
```

Comment:
why this change?

Reply:
Only that these are the defaults used by `httpx` so I changed our config object to reflect that. The `session_pool_maxsize` variable controls the total number of allowed connections while `session_pool_connections` controls the number of keepalive connections. I reckon `20` for the max allowed is quite small considering